polars/polars-io/src/csv/write.rs

```diff
@@ -100,4 +100,12 @@ where
         self.options.quote = char;
         self
     }
+
+    /// Set the CSV file's null value representation
+    pub fn with_null_value(mut self, null_value: Option<String>) -> Self {
```

Comment:
Given that we ignore the None, I think we should not accept an option here.

Reply:
Good point :) 
Tweaked accordingly...